assets/src/styles/main.scss

```diff
@@ -32,7 +45,19 @@ body {
     &#site-location {
         @import './pages/site-location';
     }
+
+    /* style the well diagram title */
+    .well-diagram-title {
+        text-align: center;
+    }
+    /* don't show the well diagram title except at narrow screen widths (where the well diagram takes the full width of the page) */
+    @media (min-width: 480px) {
```

Comment:
Can we use the theme breakpoint variable rather than hard coding 480?

Reply:
Yup. Can do.